spec/system/provider_interface/provider_can_view_interview_schedule_spec.rb

```diff
@@ -17,6 +17,13 @@
     given_i_am_a_provider_user
     and_i_sign_in_to_the_provider_interface
 
+    when_i_visit_the_provider_interface
+    and_i_click_interview_schedule
+    then_i_see_no_upcoming_interviews
+
+    and_i_click_past_interviews
+    then_i_see_no_past_interviews
+
     given_there_are_past_and_present_interviews
```

Comment:
Unrelated to this PR: I was wondering around having multiple givens in a scenario and what people's thoughts were around this as this seems to have been an adopted pattern in the specs?